Template/task_gantt/show.php

```diff
@@ -12,14 +12,19 @@
             <li id="gantt-mode-month">
                 <?php echo t('Month'); ?>
             </li>
+            <li>
+                <label id="hide-first-column-label"><input type="checkbox" value="<?= $firstColumnId ?>" id="hide-first-column">&nbsp;<?= t('Hide first Column') ?></label>
+            </li>
         </ul>
     </div>
     <?php if (!empty($tasks)) { ?>
-        <div        
-            id="gantt-chart"
-            data-save-url="<?php echo $this->url->href('TaskGanttController', 'save', ['project_id' => $project['id'], 'plugin' => 'Gantt']); ?>"
-            data-records='<?php echo json_encode($tasks); ?>'
-        ></div>        
+        <div id="gantt-chart"
+             data-save-url="<?php echo $this->url->href('TaskGanttController', 'save', ['project_id' => $project['id'], 'plugin' => 'Gantt']); ?>"
+             data-records='<?php echo json_encode($tasks); ?>'
+        ></div>
+        <div id="gantt-popup-template" class="hide">
+            <?= $this->render('Gantt:task_gantt/popup', array('project_id' => $project['id'], 'tasks' => $tasks)) ?>
```

Comment:
Should be `'project' => $project` or the hook has to use project_id

Reply:
My Bad ! It's a mistake ! :( 
However I used `$projet` var in popup.php and not `$project_id`... 🤦‍♂️
Correction : 96f55eeed68ee066b46a496111370018e3d5257d